src/marqo/tensor_search/enums.py

```diff
@@ -101,7 +101,7 @@ class EnvVars:
     MARQO_ENABLE_THROTTLING = "MARQO_ENABLE_THROTTLING"
     MARQO_LOG_LEVEL = "MARQO_LOG_LEVEL"
     MARQO_ROOT_PATH = "MARQO_ROOT_PATH"
-
+    EF_CONSTRUCTION_MAXIMUM_VALUE = "MARQO_EF_CONSTRUCTION_MAXIMUM_VALUE"
```

Comment:
Can we keep the name of the variable and the value identical (in line with the other env vars?). It would be good to keep the pattern unless there's a pressing reason to change it

Reply:
Also we can use `MAX` rather than `MAXIMUM` for conciseness

Reply:
Oh sorry, I only saw the pattern on the RHS.